internal/node/node_launcher.sh

```diff
@@ -129,8 +129,18 @@ for ARG in "${ALL_ARGS[@]}"; do
   esac
 done
 
+# Note: Bash does not forward termination signals to any child process when
+# running in docker so need to manually trap and forward the signals
+_term() {
+  kill -TERM "$child" 2>/dev/null
+}
+
 set +e
-"${node}" "${NODE_OPTIONS[@]}" "${script}" "${ARGS[@]}"
+"${node}" "${NODE_OPTIONS[@]}" "${script}" "${ARGS[@]}" &
+child=$!
+trap _term SIGTERM
+trap _term SIGINT
```

Comment:
For correctness, could you preserve the same signal? this sends a sigterm to node when the user sent a sigint, and it's possible those have different behavior under some circumstance